### PR TITLE
Create intellij_idea_set_up.md

### PR DESCRIPTION
**Pull request by val-fed**

### CODE REVIEW

File: intellij_idea_set_up.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# IntelliJ IDEA settings
+
+### Installing the Bazel plugin
+Install it from within the IDE (`Settings > Plugins > Marketplace`, and search for **Bazel**).
+  
+  See another option to install the Bazel plugin - https://ij.bazel.build/docs/bazel-plugin.html#getting-started.
+  
+  At the time of writing the plugin is compatible with IntelliJ 2017.3.3 and later.
```
Comment (oferb) [inline intellij_idea_set_up.md]: `At the time of writing->As of <date>,`

File: intellij_idea_set_up.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# IntelliJ IDEA settings
+
+### Installing the Bazel plugin
+Install it from within the IDE (`Settings > Plugins > Marketplace`, and search for **Bazel**).
+  
+  See another option to install the Bazel plugin - https://ij.bazel.build/docs/bazel-plugin.html#getting-started.
```
Comment (oferb) [inline intellij_idea_set_up.md]: `See another option to install the Bazel plugin->If that doesn't work, see alternative option`

File: intellij_idea_set_up.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- First, you need to select an existing workspace root (the directory containing the WORKSPACE file).
+![Screenshot 1](https://imgur.com/WqwODKI.png)
+- Next, you need to choose `Create from scratch` option
+![Screenshot 2](https://imgur.com/WVMThTo.png)
+- Click Next then Finish to import the project into the IDE.
+
+See more import options - https://ij.bazel.build/docs/import-project.html
+
+### Building a Project
+Run `./compile.sh build` from IDE terminal if you get a mistake during building process in the previous step.
```
Comment (oferb) [inline intellij_idea_set_up.md]: How does that solve the mistake? Or is it a workaround?

Comment (val-fed) [reply intellij_idea_set_up.md]: It is the workaround.

File: intellij_idea_set_up.md (hunk 2 above)
Comment (val-fed) [reply intellij_idea_set_up.md]: Done.

File: intellij_idea_set_up.md (hunk 1 above)
Comment (val-fed) [reply intellij_idea_set_up.md]: Done.

File: intellij_idea_set_up.md (hunk 3 above)
Comment (oferb) [reply intellij_idea_set_up.md]: ### Building a Project
Run `./compile.sh build` from IDE terminal if you get a mistake during building process in the previous step.

### Troubleshooting
If the project doesn't build, you can try running `./compile.sh build` from IDE terminal.

**Pull request merged**